src/main/scala/beam/calibration/utils/CreateExperiment.scala

```diff
@@ -1,19 +1,24 @@
 package beam.calibration.utils
 
+import java.io.File
+
 import beam.calibration.SigoptExperimentData
+import beam.experiment.{ExperimentApp, ExperimentDef}
 import com.sigopt.Sigopt
 import com.typesafe.scalalogging.LazyLogging
 
-object CreateExperiment extends LazyLogging {
+object CreateExperiment extends ExperimentApp {
   private val NEW_EXPERIMENT_FLAG = "00000"
 
   // Store CLI inputs as private members
   Sigopt.clientToken = SigOptApiToken.getClientAPIToken
 
-  def main(args: Array[String]): Unit = {
+  override def main(args: Array[String]): Unit = {
```

Comment:
@sfwatergit Any reason you made this override main? This yields a warning for a reason - I'm not sure what the end goal is here? Did you want to override everything in the original App?